v2/services/authorize/client_test.go

```diff
@@ -140,7 +141,7 @@ func Test_Loadbalancing(t *testing.T) {
 		server, err := authMock.NewServerOnHostPort(ip, serverPort)
 		require.NoError(t, err)
 		dnsServer.AddEndpoint(domain, ip)
-		server.On("DeepPing", mock.Anything, mock.Anything).Return(&common.PrimitiveString{Value: ""}, nil).Once()
+		server.On("DeepPing", mock.Anything, mock.Anything).Return(&common.PrimitiveString{Value: ""}, nil).Maybe()
```

Comment:
Seems like this is very flaky. Can't really understand why this would fail based on the current change